OrbitQt/orbitmainwindow.cpp

```diff
@@ -743,10 +769,13 @@ outcome::result<void> OrbitMainWindow::OpenCapture(
 
   if (result.has_error()) {
     SetTitle({});
-    QMessageBox::critical(this, "Error loading capture",
-                          QString::fromStdString(absl::StrFormat(
-                              "Could not load capture from \"%s\":\n%s.",
-                              filepath, result.error().message())));
+    QMessageBox::critical(
+        this, "Error loading capture",
+        QString::fromStdString(absl::StrFormat(
+            "Could not load capture from \"%s\":\n%s.\nNote: If capture was "
+            "taken in previous Orbit version, it could be unavailable. Please "
```

Comment:
Nit: "taken in a previous Orbit version" (add "a"). 

And change "unavailable" to "incompatible".